.circleci/config.yml

```diff
@@ -858,7 +858,7 @@ jobs:
             cd portafly
             yarn --version
             yarn install --frozen-lockfile
-            yarn build
+            CI=false yarn build
```

Comment:
Isn't there any other way to disable that behaviour? I'm not sure about this, specially when we don't know what else uses the `CI` env...

Reply:
Nothing else should be affected since the env var is changed for this line only, isn't it?

Reply:
In theory a build shouldn't depend on `CI`, that's true, but we are triggering from circle and since the problem is that warnings are been treated as errors, I'd expect a different flag/config to adjust the level, not completely disabled it

Reply:
I can't disagree but it's way it is 🤷‍♂️ 
More info: https://github.com/facebook/create-react-app/issues/3657

I don't like having warnings anyway, maybe we should linter-treat them as errors and use `eslint-ignore` more often 🤷‍♂️

Reply:
We should, yeah. Will leave the circle official example here for future reference:
https://circleci.com/docs/2.0/sample-config/

Reply:
OK PR coming soon!